wsclient/src/main/java/org/hpccsystems/ws/client/antlr/Ecl.g4

```diff
@@ -0,0 +1,53 @@
+grammar Ecl;		
```

Comment:
What's the likelihood of expanding this grammar to describe more ECL?
Also, in older versions of ANTLR there was a concept of importing subgramars into higher level grammars, if for some reason we end up parsing more ECL, this could be treated as the ecl record definition subgrammar
It makes sense to me to label th